service/systemd/service.go

```diff
@@ -23,11 +23,13 @@ var (
 
 	renderer = shell.BashRenderer{}
 	cmds     = commands{renderer, executable}
+
+	SystemPath = "/run/systemd/system"
```

Comment:
Can this be a constant?

```
const ystemPath = "/run/systemd/system"
```

Reply:
One of the tests needs to patch the value.